src/main/resources/openshift/deploy.sh

```diff
@@ -39,18 +39,32 @@ rc=$?; if [[ $rc != 0 ]]; then echo "Missing deployment. Please build and copy r
 echo
 echo "Openshift project"
 echo "  -> Create Openshift project (${OCP_PROJECT})"
-oc new-project ${OCP_PROJECT} 2>/dev/null > /dev/null
+oc new-project ${OCP_PROJECT} > /dev/null
 sleep 1
-
-oc policy add-role-to-user view system:serviceaccount:$(oc project -q):eap-service-account -n $(oc project -q)
-oc policy add-role-to-user view system:serviceaccount:$(oc project -q):sso-service-account -n $(oc project -q)
-
 echo "  -> Switch to project"
-oc project ${OCP_PROJECT}  2>&1 > /dev/null
+oc project ${OCP_PROJECT} > /dev/null
+sleep 1
+echo "  -> Register service accounts"
+oc policy add-role-to-user view system:serviceaccount:${OCP_PROJECT}:eap-service-account -n ${OCP_PROJECT}
+oc policy add-role-to-user view system:serviceaccount:${OCP_PROJECT}:sso-service-account -n ${OCP_PROJECT}
+sleep 1
+echo "  -> Verify ImageStreams"
+sleep 1
+M=0
+until [ ${M} -ge 50 ]
```

Comment:
Please help me to understand what is going on here. 

From my reading of it, the code seems to loop up to 50 times while searching for the middleware image streams. If the middleware streams are not available, then it keeps going anyway without logging anything. If they are available, then it keeps going. I'm not sure how this helps?